### ISSUE TITLE
[Bug]: [benchmark] milvus restart, load fails, and reports: "has not been loaded to memory or load failed"

### ISSUE THREAD

**Issue by elstic** [milvus-io/milvus#23863]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Environment

```markdown
- Milvus version: master-20230428-9d35a5f9
- Deployment mode(standalone or cluster):standalone
- MQ type(rocksmq, pulsar or kafka):    
- SDK version(e.g. pymilvus v2.0.0rc2):
- OS(Ubuntu or CentOS): 
- CPU/Memory: 
- GPU: 
- Others:
```

### Current Behavior

case: test_concurrent_locust_100m_ivf_sq8_ddl_dql_standalone
argo task : fouramf-2hasv-829xf

milvus reboot, get_loading_progress interface reports failure with "Retry run out of 10 retry times, message=collection 441108106776150240 has not been loaded to memory or load failed"

server init (14:58:44) : 
```
fouramf-2hasv-829xf-17-5288-etcd-0                                1/1     Running            0                114s    10.104.23.85    4am-node27   <none>           <none>
fouramf-2hasv-829xf-17-5288-milvus-standalone-7bfc85cf58-jxp6v    1/1     Running            0                114s    10.104.21.77    4am-node24   <none>           <none>
fouramf-2hasv-829xf-17-5288-minio-5d9c9b8f5f-srv59                1/1     Running            0                114s    10.104.23.84    4am-node27   <none>           <none>
```

Released before server (2023-04-28 16:26:47)：
```
fouramf-2hasv-829xf-17-5288-etcd-0                                1/1     Running            0                89m     10.104.23.85    4am-node27   <none>           <none>
fouramf-2hasv-829xf-17-5288-milvus-standalone-7bfc85cf58-jxp6v    1/1     Running            1 (2m41s ago)    89m     10.104.21.77    4am-node24   <none>           <none>
fouramf-2hasv-829xf-17-5288-minio-5d9c9b8f5f-srv59                1/1     Running            0                89m     10.104.23.84    4am-node27   <none>           <none>
```

client log:
```
[2023-04-28 16:17:40,994 -  INFO - fouram]: [Base] Start inserting, ids: 99950000 - 99999999, data size: 100,000,000 (base.py:164)
[2023-04-28 16:17:42,888 -  INFO - fouram]: [Time] Collection.insert run in 1.8938s (api_request.py:41)
[2023-04-28 16:17:42,907 -  INFO - fouram]: [Base] Number of vectors in the collection(fouram_o8UDl7j2): 99900000 (base.py:318)
[2023-04-28 16:17:42,965 -  INFO - fouram]: [Base] Total time of insert: 3312.872s, average number of vector bars inserted per second: 30185.2894, average time to insert 50000 vectors per time: 1.6564s (base.py:235)
[2023-04-28 16:17:42,966 -  INFO - fouram]: [Base] Start flush collection fouram_o8UDl7j2 (base.py:133)
[2023-04-28 16:17:44,985 -  INFO - fouram]: [Base] Number of vectors in the collection(fouram_o8UDl7j2): 100000000 (base.py:318)
[2023-04-28 16:17:44,989 -  INFO - fouram]: [Base] Params of index: {'index_type': 'IVF_SQ8', 'metric_type': 'L2', 'params': {'nlist': 2048}} (base.py:296)
[2023-04-28 16:17:44,989 -  INFO - fouram]: [Base] Start build index of IVF_SQ8 for collection fouram_o8UDl7j2, params:{'index_type': 'IVF_SQ8', 'metric_type': 'L2', 'params': {'nlist': 2048}} (base.py:283)
[2023-04-28 16:18:59,835 -  INFO - fouram]: [Time] Index run in 74.8445s (api_request.py:41)
[2023-04-28 16:18:59,836 -  INFO - fouram]: [CommonCases] RT of build index IVF_SQ8: 74.8445s (common_cases.py:87)
[2023-04-28 16:18:59,839 -  INFO - fouram]: [Base] Params of index: {'index_type': 'IVF_SQ8', 'metric_type': 'L2', 'params': {'nlist': 2048}} (base.py:296)
[2023-04-28 16:18:59,839 -  INFO - fouram]: [CommonCases] Prepare index IVF_SQ8 done. (common_cases.py:90)
[2023-04-28 16:18:59,839 -  INFO - fouram]: [CommonCases] No scalars need to be indexed. (common_cases.py:95)
[2023-04-28 16:18:59,839 -  INFO - fouram]: [Base] Start load collection fouram_o8UDl7j2,replica_number:1,kwargs:{} (base.py:139)
[2023-04-28 16:24:06,256 - WARNING - fouram]: [93m[get_loading_progress] retry:4, cost: 0.27s, reason: <_MultiThreadedRendezvous: StatusCode.UNAVAILABLE, failed to connect to all addresses; last error: UNKNOWN: ipv4:10.255.142.184:19530: Failed to connect to remote host: Connection refused>[0m (decorators.py:71)
[2023-04-28 16:24:06,528 - WARNING - fouram]: [93m[get_loading_progress] retry:5, cost: 0.81s, reason: <_MultiThreadedRendezvous: StatusCode.UNAVAILABLE, failed to connect to all addresses; last error: UNKNOWN: ipv4:10.255.142.184:19530: Failed to connect to remote host: Connection refused>[0m (decorators.py:71)
[2023-04-28 16:24:07,339 - WARNING - fouram]: [93m[get_loading_progress] retry:6, cost: 2.43s, reason: <_MultiThreadedRendezvous: StatusCode.UNAVAILABLE, failed to connect to all addresses; last error: UNKNOWN: ipv4:10.255.142.184:19530: Failed to connect to remote host: Connection refused>[0m (decorators.py:71)
[2023-04-28 16:24:09,773 - WARNING - fouram]: [93m[get_loading_progress] retry:7, cost: 7.29s, reason: <_MultiThreadedRendezvous: StatusCode.UNAVAILABLE, failed to connect to all addresses; last error: UNKNOWN: ipv4:10.255.142.184:19530: Failed to connect to remote host: Connection refused>[0m (decorators.py:71)
[2023-04-28 16:24:17,072 - WARNING - fouram]: [93m[get_loading_progress] retry:8, cost: 21.87s, reason: <_MultiThreadedRendezvous: StatusCode.UNAVAILABLE, failed to connect to all addresses; last error: UNKNOWN: ipv4:10.255.142.184:19530: Failed to connect to remote host: Connection refused>[0m (decorators.py:71)
[2023-04-28 16:24:38,964 - WARNING - fouram]: [93m[get_loading_progress] retry:9, cost: 60.00s, reason: <_MultiThreadedRendezvous: StatusCode.UNAVAILABLE, failed to connect to all addresses; last error: UNKNOWN: ipv4:10.255.142.184:19530: Failed to connect to remote host: Connection refused>[0m (decorators.py:71)
[2023-04-28 16:25:39,022 - WARNING - fouram]: [93m[get_loading_progress] retry:10, cost: 60.00s, reason: <_MultiThreadedRendezvous: StatusCode.UNAVAILABLE, failed to connect to all addresses; last error: UNKNOWN: ipv4:10.255.142.184:19530: Failed to connect to remote host: Connection refused>[0m (decorators.py:71)
[2023-04-28 16:26:39,087 - WARNING - fouram]: [93mRetry run out of 10 retry times[0m (decorators.py:79)
[2023-04-28 16:26:39,089 - ERROR - fouram]: RPC error: [get_loading_progress], <MilvusException: (code=1, message=Retry run out of 10 retry times, message=collection 441108106776150240 has not been loaded to memory or load failed)>, <Time:{'RPC start': '2023-04-28 16:23:56.865313', 'RPC error': '2023-04-28 16:26:39.089310'}> (decorators.py:108)
[2023-04-28 16:26:39,090 - ERROR - fouram]: RPC error: [wait_for_loading_collection], <MilvusException: (code=1, message=Retry run out of 10 retry times, message=collection 441108106776150240 has not been loaded to memory or load failed)>, <Time:{'RPC start': '2023-04-28 16:18:59.854749', 'RPC error': '2023-04-28 16:26:39.090011'}> (decorators.py:108)
[2023-04-28 16:26:39,090 - ERROR - fouram]: RPC error: [load_collection], <MilvusException: (code=1, message=Retry run out of 10 retry times, message=collection 441108106776150240 has not been loaded to memory or load failed)>, <Time:{'RPC start': '2023-04-28 16:18:59.840116', 'RPC error': '2023-04-28 16:26:39.090137'}> (decorators.py:108)
[2023-04-28 16:26:39,091 - ERROR - fouram]: (api_response) : <MilvusException: (code=1, message=Retry run out of 10 retry times, message=collection 441108106776150240 has not been loaded to memory or load failed)> (api_request.py:49)
[2023-04-28 16:26:39,091 - ERROR - fouram]: [CheckFunc] load request check failed, response:<MilvusException: (code=1, message=Retry run out of 10 retry times, message=collection 441108106776150240 has not been loaded to memory or load failed)> (func_check.py:52)

```

client pod: fouramf-2hasv-829xf-3519293898

### Expected Behavior

load success. 

### Steps To Reproduce

```markdown
1. create a collection or use an existing collection
        2. build index on vector column
        3. insert a certain number of vectors
        4. flush collection
        5. build index on vector column with the same parameters
        6. build index on on scalars column or not
        7. count the total number of rows
        8. load collection  ==> failed
        9. perform concurrent operations
        10. clean all collections or not
```

**Comment by elstic** [comment]
### querynoode crash and load failed 

case: test_concurrent_locust_100m_hnsw_ddl_dql_filter_kafka_cluster
argo task: fouramf-hdvbk

server 10:57:52
```
fouramf-hdvbk-91-6542-etcd-0                                      1/1     Running     0                115s    10.104.24.235   4am-node29   <none>           <none>
fouramf-hdvbk-91-6542-etcd-1                                      1/1     Running     0                114s    10.104.21.158   4am-node24   <none>           <none>
fouramf-hdvbk-91-6542-etcd-2                                      1/1     Running     0                114s    10.104.20.118   4am-node22   <none>           <none>
fouramf-hdvbk-91-6542-kafka-0                                     1/1     Running     1 (80s ago)      115s    10.104.20.116   4am-node22   <none>           <none>
fouramf-hdvbk-91-6542-kafka-1                                     1/1     Running     0                114s    10.104.4.90     4am-node11   <none>           <none>
fouramf-hdvbk-91-6542-kafka-2                                     1/1     Running     1 (74s ago)      114s    10.104.16.181   4am-node21   <none>           <none>
fouramf-hdvbk-91-6542-milvus-datacoord-6fc5495d79-ktvzt           1/1     Running     0                115s    10.104.16.175   4am-node21   <none>           <none>
fouramf-hdvbk-91-6542-milvus-datanode-67b87776c6-zjzpn            1/1     Running     0                115s    10.104.16.176   4am-node21   <none>           <none>
fouramf-hdvbk-91-6542-milvus-indexcoord-cf9645b69-tz287           1/1     Running     0                114s    10.104.24.231   4am-node29   <none>           <none>
fouramf-hdvbk-91-6542-milvus-indexnode-865594d7d-mn5rj            1/1     Running     0                115s    10.104.6.145    4am-node13   <none>           <none>
fouramf-hdvbk-91-6542-milvus-proxy-5df68b76b7-xmzpb               1/1     Running     0                114s    10.104.24.232   4am-node29   <none>           <none>
fouramf-hdvbk-91-6542-milvus-querycoord-8555bcdccd-hlgch          1/1     Running     0                114s    10.104.24.230   4am-node29   <none>           <none>
fouramf-hdvbk-91-6542-milvus-querynode-59844fddd6-8znxd           1/1     Running     0                115s    10.104.15.41    4am-node20   <none>           <none>
fouramf-hdvbk-91-6542-milvus-querynode-59844fddd6-xctmd           1/1     Running     0                115s    10.104.23.154   4am-node27   <none>           <none>
fouramf-hdvbk-91-6542-milvus-rootcoord-77fb547d98-jz5p9           1/1     Running     0                115s    10.104.24.233   4am-node29   <none>           <none>
fouramf-hdvbk-91-6542-minio-0                                     1/1     Running     0                115s    10.104.24.234   4am-node29   <none>           <none>
fouramf-hdvbk-91-6542-minio-1                                     1/1     Running     0                114s    10.104.20.117   4am-node22   <none>           <none>
fouramf-hdvbk-91-6542-minio-2                                     1/1     Running     0                114s    10.104.21.150   4am-node24   <none>           <none>
fouramf-hdvbk-91-6542-minio-3                                     1/1     Running     0                114s    10.104.4.92     4am-node11   <none>           <none>
fouramf-hdvbk-91-6542-zookeeper-0                                 1/1     Running     0                115s    10.104.16.178   4am-node21   <none>           <none>
fouramf-hdvbk-91-6542-zookeeper-1                                 1/1     Running     0                114s    10.104.21.162   4am-node24   <none>           <none>
fouramf-hdvbk-91-6542-zookeeper-2                                 1/1     Running     0                114s    10.104.18.64    4am-node25   <none>           <none>
```
server 14:06:01:
```
fouramf-hdvbk-91-6542-etcd-0                                      1/1     Running            0                 3h10m   10.104.24.235   4am-node29   <none>           <none>
fouramf-hdvbk-91-6542-etcd-1                                      1/1     Running            0                 3h10m   10.104.21.158   4am-node24   <none>           <none>
fouramf-hdvbk-91-6542-etcd-2                                      1/1     Running            0                 3h10m   10.104.20.118   4am-node22   <none>           <none>
fouramf-hdvbk-91-6542-kafka-0                                     1/1     Running            1 (3h9m ago)      3h10m   10.104.20.116   4am-node22   <none>           <none>
fouramf-hdvbk-91-6542-kafka-1                                     1/1     Running            0                 3h10m   10.104.4.90     4am-node11   <none>           <none>
fouramf-hdvbk-91-6542-kafka-2                                     1/1     Running            1 (3h9m ago)      3h10m   10.104.16.181   4am-node21   <none>           <none>
fouramf-hdvbk-91-6542-milvus-datacoord-6fc5495d79-ktvzt           1/1     Running            0                 3h10m   10.104.16.175   4am-node21   <none>           <none>
fouramf-hdvbk-91-6542-milvus-datanode-67b87776c6-zjzpn            1/1     Running            0                 3h10m   10.104.16.176   4am-node21   <none>           <none>
fouramf-hdvbk-91-6542-milvus-indexcoord-cf9645b69-tz287           1/1     Running            0                 3h10m   10.104.24.231   4am-node29   <none>           <none>
fouramf-hdvbk-91-6542-milvus-indexnode-865594d7d-mn5rj            1/1     Running            0                 3h10m   10.104.6.145    4am-node13   <none>           <none>
fouramf-hdvbk-91-6542-milvus-proxy-5df68b76b7-xmzpb               1/1     Running            0                 3h10m   10.104.24.232   4am-node29   <none>           <none>
fouramf-hdvbk-91-6542-milvus-querycoord-8555bcdccd-hlgch          1/1     Running            0                 3h10m   10.104.24.230   4am-node29   <none>           <none>
fouramf-hdvbk-91-6542-milvus-querynode-59844fddd6-8znxd           0/1     CrashLoopBackOff   4 (18s ago)       3h10m   10.104.15.41    4am-node20   <none>           <none>
fouramf-hdvbk-91-6542-milvus-querynode-59844fddd6-xctmd           0/1     Running            4 (2m ago)        3h10m   10.104.23.154   4am-node27   <none>           <none>
fouramf-hdvbk-91-6542-milvus-rootcoord-77fb547d98-jz5p9           1/1     Running            0                 3h10m   10.104.24.233   4am-node29   <none>           <none>
fouramf-hdvbk-91-6542-minio-0                                     1/1     Running            0                 3h10m   10.104.24.234   4am-node29   <none>           <none>
fouramf-hdvbk-91-6542-minio-1                                     1/1     Running            0                 3h10m   10.104.20.117   4am-node22   <none>           <none>
fouramf-hdvbk-91-6542-minio-2                                     1/1     Running            0                 3h10m   10.104.21.150   4am-node24   <none>           <none>
fouramf-hdvbk-91-6542-minio-3                                     1/1     Running            0                 3h10m   10.104.4.92     4am-node11   <none>           <none>
fouramf-hdvbk-91-6542-zookeeper-0                                 1/1     Running            0                 3h10m   10.104.16.178   4am-node21   <none>           <none>
fouramf-hdvbk-91-6542-zookeeper-1                                 1/1     Running            0                 3h10m   10.104.21.162   4am-node24   <none>           <none>
fouramf-hdvbk-91-6542-zookeeper-2                                 1/1     Running            0                 3h10m   10.104.18.64    4am-node25   <none>           <none>
```


client log:
```
[2023-05-06 12:05:36,741 -  INFO - fouram]: [Base] Number of vectors in the collection(fouram_E3Irobqz): 99900000 (base.py:318)
[2023-05-06 12:05:36,884 -  INFO - fouram]: [Base] Start inserting, ids: 99950000 - 99999999, data size: 100,000,000 (base.py:164)
[2023-05-06 12:05:38,288 -  INFO - fouram]: [Time] Collection.insert run in 1.4035s (api_request.py:41)
[2023-05-06 12:05:38,291 -  INFO - fouram]: [Base] Number of vectors in the collection(fouram_E3Irobqz): 99900000 (base.py:318)
[2023-05-06 12:05:38,343 -  INFO - fouram]: [Base] Total time of insert: 2941.9965s, average number of vector bars inserted per second: 33990.5231, average time to insert 50000 vectors per time: 1.471s (base.py:235)
[2023-05-06 12:05:38,343 -  INFO - fouram]: [Base] Start flush collection fouram_E3Irobqz (base.py:133)
[2023-05-06 12:05:40,866 -  INFO - fouram]: [Base] Number of vectors in the collection(fouram_E3Irobqz): 100000000 (base.py:318)
[2023-05-06 12:05:40,870 -  INFO - fouram]: [Base] Params of index: {'index_type': 'HNSW', 'metric_type': 'L2', 'params': {'M': 8, 'efConstruction': 200}} (base.py:296)
[2023-05-06 12:05:40,871 -  INFO - fouram]: [Base] Start build index of HNSW for collection fouram_E3Irobqz, params:{'index_type': 'HNSW', 'metric_type': 'L2', 'params': {'M': 8, 'efConstruction': 200}} (base.py:283)
[2023-05-06 13:54:06,155 -  INFO - fouram]: [Time] Index run in 6505.2836s (api_request.py:41)
[2023-05-06 13:54:06,157 -  INFO - fouram]: [CommonCases] RT of build index HNSW: 6505.2836s (common_cases.py:87)
[2023-05-06 13:54:06,161 -  INFO - fouram]: [Base] Params of index: {'index_type': 'HNSW', 'metric_type': 'L2', 'params': {'M': 8, 'efConstruction': 200}} (base.py:296)
[2023-05-06 13:54:06,161 -  INFO - fouram]: [CommonCases] Prepare index HNSW done. (common_cases.py:90)
[2023-05-06 13:54:06,161 -  INFO - fouram]: [CommonCases] No scalars need to be indexed. (common_cases.py:95)
[2023-05-06 13:54:06,162 -  INFO - fouram]: [Base] Start load collection fouram_E3Irobqz,replica_number:1,kwargs:{} (base.py:139)
[2023-05-06 14:06:01,147 - ERROR - fouram]: RPC error: [get_loading_progress], <MilvusException: (code=52, message=deny to load, insufficient memory, please allocate more resources, collectionName: fouram_E3Irobqz)>, <Time:{'RPC start': '2023-05-06 14:06:01.145520', 'RPC error': '2023-05-06 14:06:01.147498'}> (decorators.py:108)
[2023-05-06 14:06:01,150 - ERROR - fouram]: RPC error: [wait_for_loading_collection], <MilvusException: (code=52, message=deny to load, insufficient memory, please allocate more resources, collectionName: fouram_E3Irobqz)>, <Time:{'RPC start': '2023-05-06 13:54:06.193802', 'RPC error': '2023-05-06 14:06:01.150341'}> (decorators.py:108)
[2023-05-06 14:06:01,150 - ERROR - fouram]: RPC error: [load_collection], <MilvusException: (code=52, message=deny to load, insufficient memory, please allocate more resources, collectionName: fouram_E3Irobqz)>, <Time:{'RPC start': '2023-05-06 13:54:06.162191', 'RPC error': '2023-05-06 14:06:01.150454'}> (decorators.py:108)
[2023-05-06 14:06:01,151 - ERROR - fouram]: (api_response) : <MilvusException: (code=52, message=deny to load, insufficient memory, please allocate more resources, collectionName: fouram_E3Irobqz)> (api_request.py:49)
[2023-05-06 14:06:01,151 - ERROR - fouram]: [CheckFunc] load request check failed, response:<MilvusException: (code=52, message=deny to load, insufficient memory, please allocate more resources, collectionName: fouram_E3Irobqz)>
```

memory usage:
![image](https://user-images.githubusercontent.com/48523564/236720904-af8351f4-dc4d-49f5-872e-2e889676d60f.png)

**Comment by xiaofan-luan** [comment]
it seesm not to be a load issue
st error: UNKNOWN: ipv4:10.255.142.184:19530: Failed to connect to remote host: Connection refused

and the second issue seems to be insufficient memory? not really related

**Comment by elstic** [comment]
> it seesm not to be a load issue st error: UNKNOWN: ipv4:10.255.142.184:19530: Failed to connect to remote host: Connection refused
> 
> and the second issue seems to be insufficient memory? not really related

The second issue, which is a frequently run case , is that we limit memory to 64G for querynode, which is usually enough, and the same case does not have this problem in the 2.2.0 branch.

**Comment by sunby** [comment]
This issue is fixed by https://github.com/milvus-io/milvus/pull/23968
Please verify it @elstic

**Comment by elstic** [comment]
The first problem is verified and fixed, verify image: master-20230512-113f9a0e.

The second issue, querynoode crash, no longer exists, but there are insert data failures and deny to write.

image: master-20230512-113f9a0e 
case: test_concurrent_locust_100m_hnsw_ddl_dql_filter_kafka_cluster


server:
```
qtp-issue23863-dbruj-etcd-0                                       1/1     Running            0                 2m27s   10.104.21.206   4am-node24   <none>           <none>
qtp-issue23863-dbruj-etcd-1                                       1/1     Running            0                 2m27s   10.104.5.98     4am-node12   <none>           <none>
qtp-issue23863-dbruj-etcd-2                                       1/1     Running            0                 2m27s   10.104.22.151   4am-node26   <none>           <none>
qtp-issue23863-dbruj-kafka-0                                      1/1     Running            1 (2m2s ago)      2m27s   10.104.21.207   4am-node24   <none>           <none>
qtp-issue23863-dbruj-kafka-1                                      1/1     Running            1 (118s ago)      2m27s   10.104.22.149   4am-node26   <none>           <none>
qtp-issue23863-dbruj-kafka-2                                      1/1     Running            0                 2m27s   10.104.5.102    4am-node12   <none>           <none>
qtp-issue23863-dbruj-milvus-datacoord-5798477cb-k4nzr             1/1     Running            0                 2m27s   10.104.6.144    4am-node13   <none>           <none>
qtp-issue23863-dbruj-milvus-datanode-859fdb5bbc-xqjcq             1/1     Running            0                 2m27s   10.104.21.203   4am-node24   <none>           <none>
qtp-issue23863-dbruj-milvus-indexcoord-7897654696-bh8kt           1/1     Running            0                 2m27s   10.104.17.19    4am-node23   <none>           <none>
qtp-issue23863-dbruj-milvus-indexnode-77d646c6f9-98g7j            1/1     Running            0                 2m27s   10.104.6.146    4am-node13   <none>           <none>
qtp-issue23863-dbruj-milvus-proxy-8658f4d88b-w8xwp                1/1     Running            0                 2m27s   10.104.5.94     4am-node12   <none>           <none>
qtp-issue23863-dbruj-milvus-querycoord-54b487b6b7-vrbsz           1/1     Running            0                 2m27s   10.104.6.147    4am-node13   <none>           <none>
qtp-issue23863-dbruj-milvus-querynode-7975fc9fff-g97fp            1/1     Running            0                 2m27s   10.104.6.145    4am-node13   <none>           <none>
qtp-issue23863-dbruj-milvus-querynode-7975fc9fff-h94lr            1/1     Running            0                 2m27s   10.104.9.253    4am-node14   <none>           <none>
qtp-issue23863-dbruj-milvus-rootcoord-5654c65759-9qccj            1/1     Running            0                 2m27s   10.104.21.202   4am-node24   <none>           <none>
qtp-issue23863-dbruj-minio-0                                      1/1     Running            0                 2m27s   10.104.9.2      4am-node14   <none>           <none>
qtp-issue23863-dbruj-minio-1                                      1/1     Running            0                 2m27s   10.104.5.101    4am-node12   <none>           <none>
qtp-issue23863-dbruj-minio-2                                      1/1     Running            0                 2m27s   10.104.22.152   4am-node26   <none>           <none>
qtp-issue23863-dbruj-minio-3                                      1/1     Running            0                 2m27s   10.104.19.22    4am-node28   <none>           <none>
qtp-issue23863-dbruj-zookeeper-0                                  1/1     Running            0                 2m27s   10.104.17.21    4am-node23   <none>           <none>
qtp-issue23863-dbruj-zookeeper-1                                  1/1     Running            0                 2m27s   10.104.19.21    4am-node28   <none>           <none>
qtp-issue23863-dbruj-zookeeper-2                                  1/1     Running            0                 2m27s   10.104.5.97     4am-node12   <none>           <none>
```

client error log:
`[batch_insert], <MilvusException: (code=55, message=deny to write, reason: time tick long delay, req: /milvus.proto.milvus.MilvusService/Insert`
![image](https://github.com/milvus-io/milvus/assets/48523564/8f894819-c64f-435b-9b68-f832c0e3bd35)


memory usage:
![image](https://github.com/milvus-io/milvus/assets/48523564/e1a8fe60-5c22-4fbd-a18b-23aecef5b943)

**Comment by elstic** [comment]
Verified and fixed. 
image: master-20230601-93ea9c49
